### PR TITLE
add instructions for java.allow_adding_patches_via_api config

### PR DESCRIPTION
**Pull request by lucidd**
# Description

add instructions for java.allow_adding_patches_via_api config

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [X] Manager-4.1
- [X] Manager-4.2
- [X] Uyuni

# Links

### CODE REVIEW

File: modules/client-configuration/pages/clients-centos.adoc [hunk 1]
```diff
@@ -193,5 +193,10 @@ export SPACEWALK_USER='<adminname>';export SPACEWALK_PASS='<password>'
 ----
 ln -s /usr/local/bin/cent-errata.sh /etc/cron.daily
 ----
+.Add the following line to you rhn.conf:
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-centos.adoc:196]:
```suggestion
. Open the ``rhn.conf`` configuration file, and add or edit this line:
```

File: modules/client-configuration/pages/clients-centos.adoc [hunk 2]
```diff
@@ -193,5 +193,10 @@ export SPACEWALK_USER='<adminname>';export SPACEWALK_PASS='<password>'
...
++
+----
+java.allow_adding_patches_via_api = centos7-x86_64-updates,centos7-x86_64,centos7-x86_64-extras
```
Comment (Loquacity) [inline modules/client-configuration/pages/clients-centos.adoc:199]: Can this be genericised for the CentOS version?

**Review by Loquacity — CHANGES_REQUESTED** [review]

**Review by jcayouette — APPROVED** [review]

File: modules/client-configuration/pages/clients-centos.adoc (hunk 2 above)
Comment (lucidd) [reply modules/client-configuration/pages/clients-centos.adoc:199]: i think for 6 and 7 but 8 might be a bit different. In any case these labels need to match the ones from --include-channels in the script from the previous script.

Comment (Loquacity) [reply modules/client-configuration/pages/clients-centos.adoc:199]: Right. Let's leave it as-is for now. We'll need to update it to 8 at some point, though.

**Review by Loquacity — APPROVED** [review]

**Review by keichwa — APPROVED** [review]

**Comment by Loquacity** [comment]
:heavy_check_mark: backported to `manager-4.1` https://github.com/uyuni-project/uyuni-docs/commit/98c0f2d82eaebaeee35a7a66d443e6bc399843a0

**Comment by moio** [comment]
Argh, I had some fixups submitted as PR to @lucidd's branch but went unnoticed. I am porting them into a new PR now.

**Comment by moio** [comment]
https://github.com/lucidd/uyuni-docs/pull/1

**Pull request merged**